apps/api-catalogue/components/InputSearch/InputSearch.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useCallback } from 'react'
+import * as styles from './InputSearch.treat'
+import cn from 'classnames'
+import { Icon, IconDeprecated } from '@island.is/island-ui/core'
+
+export interface InputSearchProps {
+  name: string
+  value?: string | number
+  id?: string
+  className?: string
+  placeholder?: string
+  loading?: boolean
+  colored?: boolean
+  onChange?: (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+  ) => void
+}
+
+export const InputSearch = (givenValues: InputSearchProps, ...props) => {
+  const [hasFocus, setHasFocus] = useState(false)
+  const onFocus = useCallback(() => setHasFocus(true), [setHasFocus])
+  const onBlur = useCallback(() => setHasFocus(false), [setHasFocus])
+
+  return (
+    <div
+      className={cn(styles.wrapper, ...props, hasFocus ? styles.focused : {})}
+    >
+      <input
+        className={cn(styles.input, givenValues.colored ? styles.colored : {})}
+        id={givenValues.id}
+        type="text"
+        value={givenValues.value}
+        placeholder={givenValues.placeholder}
+        onChange={givenValues.onChange}
+        onFocus={onFocus}
+        onBlur={onBlur}
+      />
+      <div>
+        {!givenValues.loading ? (
+          <span className={cn(styles.loadingIcon)}>
+            <Icon type="filled" icon="search" className={cn(styles.search)} />
+          </span>
+        ) : (
+          <span
+            className={cn(styles.loadingIcon)}
+            aria-hidden="false"
+            aria-label="Loading"
+          >
+            <IconDeprecated type="loading" className={styles.loading} />
```

Comment:
Probably better to use the new Icon component

Reply:
Yeah, we do absolutely agree on that. The reason for us using the IconDeprecated component for now was due to the fact that the new Icon component was throwing this error for us.
![image](https://user-images.githubusercontent.com/23635966/96704398-8e102b80-1383-11eb-9d93-77ba112be585.png)

Basically it seems like Next.js doesn't support Suspense yet. If you or anyone else knows any fixes for this that would be great, otherwise this might be something that the UI team needs to look at.

Reply:
Something @barabrian can probably help you with, he set it up, he might know what config is missing for that

Reply:
this should be fixed now on master